app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,19 @@
+class ReviewsController < ApplicationController
+
+  def new
+    @shelter = Shelter.find(params[:id])
+  end
+
+  def create
+    shelter = Shelter.find(params[:id])
+    new_review = Review.create(review_params)
+    shelter.reviews << new_review
+    redirect_to("/shelters/#{new_review.shelter_id}")
+  end
+
+  private
+
+  def review_params
```

Comment:
I had these set up too so I see we're already good to go now with our review_params